DiscordBot.DataAccess/EventsSheetsService.cs

```diff
@@ -520,7 +520,16 @@ private async Task<IEnumerable<EventResponse>> GetEventResponseOptionsAsync(int
             IEnumerable<EventResponse> responseColumns
         )
         {
-            var newRow = responseColumns
+            var responseList = responseColumns.ToList();
+
+            if (responseList.All(response => response.Emoji != responseEmoji))
```

Comment:
Minor but I'd probably find `if (!responseList.Any(...))` a bit easier to understand? 🙂

Reply:
Yeah sure! Changed in 1dfe1d6 🙂